### PR TITLE
Bugs/qa/account background not full height #142714135

### PR DESCRIPTION
**Pull request by callaghanc**
[#142714135] Refactored account routes; gave account pages grey background

### CODE REVIEW

**Comment by kaptron** [comment]
hey @callaghanc, I see that you refactored all the account pages to live under the same namespace, however it seems like the account layout.html template is not actually being used? (can point out why/what is going on there -- would also say that if we were to use that template, the `span` should be a `div` just for semantics)

So I'm curious what the actual modification was in order to correct the gray background. Depending on the setup of the refactoring, it could also mean that we could remove all the repetitive references to `controller: 'AccountController'` in each of the `account.xxx` routes.

Anyway we can take a look / chat more on Monday.

File: app/assets/javascripts/config/angularRoutes.js.coffee [hunk 1]
```diff
@@ -120,7 +120,15 @@
     ##########################
     # TODO: refactor "my account" pages to be under the same namespace/controller
```
[the remark this answers is not in the document]
Comment (callaghanc) [reply app/assets/javascripts/config/angularRoutes.js.coffee]: Oh right forgot to do that!

**Comment by callaghanc** [comment]
I'm not sure about the layout template, @chrisdolendo contributed that part - @chrisdolendo do you know more about that? I don't personally know much about Angular routing, so if you'd like to make changes to that or to the AccountController references that should be fine @kaptron. 

The actual modification needed for the grey background is the application of the class "center-body" on the body. That is assigned whenever on an account route in SharedController, where that class was already being applied whenever on a short form route.

File: app/assets/javascripts/config/angularRoutes.js.coffee [hunk 2]
```diff
@@ -120,7 +120,15 @@
...
     ############
-    .state('dahlia.create-account', {
+    .state('dahlia.account', {
+      url: '/account'
+      abstract: true
+      views:
+        'container@':
+          templateUrl: 'account/templates/layout.html'
```
Comment (chrisdolendo) [inline app/assets/javascripts/config/angularRoutes.js.coffee]: Peia Luzzi

**Comment by kaptron** [comment]
Closing this version of the PR but we can keep this branch for posterity, if we ever do back to this method of refactoring account routes. The problem was that create account on "save and finish later" was broken (visually, and submit didn't work) and at that point I figured doing all this account/route refactoring had too much at stake for just the CSS change needed, rather than trying to fix it. We still may want to return to this (putting all account routes under the same namespace) but seems overkill for this particular use case.

**Pull request closed without merging**